src/Form/EntityViewDisplayEditForm.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Drupal\fb_instant_articles\Form;
+
+use Drupal\field_ui\Form\EntityViewDisplayEditForm as CoreEntityViewDisplayEditForm;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+
+/**
+ * Extends the core EntityViewDisplayEditForm to support multiple regions.
+ */
+class EntityViewDisplayEditForm extends CoreEntityViewDisplayEditForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getTableHeader() {
```

Comment:
I added this and buildFieldRow() because the FBIA display was only showing the same columns and rows as all the other display types.  This was what it took for me to get the Regions column to show up in the FBIA display settings.